File: backend/ecoWater/ecoWater/src/main/java/com/project/ecoWater/device/app/DeviceDTO.java
package com.project.ecoWater.device.app;

import com.project.ecoWater.tank.app.TankDTO;
import com.project.ecoWater.user.app.UserDTO;
import com.project.ecoWater.user.domain.User;
import lombok.Builder;
import lombok.Data;

import java.util.UUID;

@Data
@Builder
public class DeviceDTO {
    private String deviceId;
    private UserDTO user;
    private TankDTO tank;
}

File: frontend/Proyecto/Proyecto/app/src/main/java/com/example/proyecto/model/device/DeviceDTO.kt
package com.example.proyecto.model.device

import com.example.proyecto.model.tank.TankDTO
import com.example.proyecto.model.user.UserDTO

data class DeviceDTO(
    val deviceId: String?=null,
    val user: UserDTO?=null,
    val tank: TankDTO?=null
)
